src/services/billingService.ts

```diff
@@ -16,10 +16,22 @@ enum PaymentActionType {
 
 class billingService {
     public async getPlans(): Promise<Plan[]> {
+        const token = getToken();
         try {
-            const response = await HTTPService.get(
-                `${ENDPOINT}/billing/plans/v2`
-            );
+            let response;
+            if (!token) {
```

Comment:
Will this ever be called?

Reply:
added that code for completeness...basically if in future we ever add `getPlans` without login the user , then this function would not fail